code/go/pkg/validator/validator_test.go

```diff
@@ -78,10 +78,10 @@ func TestValidateFile(t *testing.T) {
 				"field policy_templates.2.data_streams.1: data stream doesn't exist",
 			},
 		},
-		"bad_github_owner": {
+		"bad_github_owner/": {
```

Comment:
@shahzad31 is this change required?

Reply:
bad_github_owner wasn't bad anymore :) 

/ makes it bad now :)

Reply:
This is the name of the directory with the package, it should not make any difference to have the slash or not.
I would prefer not having it for consistency with the rest of packages listed here.